src/System.Text.Json/ref/System.Text.Json.cs

```diff
@@ -71,4 +89,95 @@ public enum JsonTokenType : byte
         public bool TryGetInt64Value(out long value) { throw null; }
         public bool TryGetSingleValue(out float value) { throw null; }
     }
+    public ref partial struct Utf8JsonWriter
+    {
+        private object _dummy;
+        public Utf8JsonWriter(System.Buffers.IBufferWriter<byte> bufferWriter, System.Text.Json.JsonWriterState state = default(System.Text.Json.JsonWriterState)) { throw null; }
+        public long BytesCommitted { get { throw null; } }
+        public long BytesWritten { get { throw null; } }
+        public int CurrentDepth { get { throw null; } }
+        public System.Text.Json.JsonWriterState CurrentState { get { throw null; } }
+        public void Flush(bool isFinalBlock = true) { }
+        public void WriteBoolean(System.ReadOnlySpan<byte> propertyName, bool value, bool suppressEscaping = false) { }
+        public void WriteBoolean(System.ReadOnlySpan<char> propertyName, bool value, bool suppressEscaping = false) { }
```

Comment:
Isn't the double negative of supressEscaping a bit strange?  I know we generally prefer default arguments to be false rather than true, but it seems like this would be better as `bool escape = true` or `bool escapeIfNecessary = true` or some such thing.

Reply:
Given we already have `isFinalBlock = true` as the default argument in Flush(), I am going to make this change as well.